pubspec.yaml

```
@@ -0,0 +1,206 @@
+  bcrypt: ^1.1.3
+  # flutter_phoenix: ^1.1.0
+  rebirth: ^1.0.2
+  filesystem_picker: ^4.1.0
+  # user_inactivity_detector: ^1.0.4
+  # For filtering json files
+  path: ^1.8.2
+  get_it: ^7.2.0
+  another_flushbar: ^1.12.29
+  keybinder: ^1.0.1
+  dotted_line: ^3.1.0
+  # meta: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)